### PR TITLE
Layout fix

### PR DESCRIPTION
**Pull request by carriemorrison**
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

---
Converted page layout to use flex and use sticky position logic, due to column lengths I also added some minimal page JS which will handle changing the sticky logic to the bottom when needed. Also added Fixed header.

flex3

flex

flex2

### Types of changes

What types of changes does your code introduce? Put an x in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

Review the following points, and put an x in all the boxes that apply. If you’re unsure about any of these, don’t hesitate to ask. We’re here to help!

- [ ] I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING](https://github.com/acss-io/atomizer/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by carriemorrison** [comment]
Wait just found 1 issue, fixing...

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -12,19 +12,14 @@
...
-                    {{ content }}
-                </div>
-                {%- include footer.html -%}
-            </div>
-        </div>
+    <body class="Pt(3.2em)">
+        {%- include header.html -%}
+        {{ content }}
+        {%- include footer.html -%}
+        <script src="{{'/assets/js/page.js' | relative_url }}"></script>
```
Comment (redonkulus) [inline docs/_layouts/default.html:22]: nit: we have been putting the scripts below

**Comment by carriemorrison** [comment]
fs

Fixed the nav too - the page does get reset to scroll top but a user is going to use the nav 99% of the time so not much of an issue...

nav

**Review by redonkulus — APPROVED** [review]
This is great! Nice to clean some of the old styles.

**Pull request merged**